ecs-cli/modules/commands/flags/flags.go

```diff
@@ -193,6 +192,26 @@ func OptionalRegionAndProfileFlags() []cli.Flag {
 	}
 }
 
+func OptClusterConfigFlag() []cli.Flag {
+	return []cli.Flag{
+		cli.StringFlag{
+			Name: ClusterConfigFlag,
+			Usage: fmt.Sprintf(
+				"[Optional] Specifies the name of the ECS cluster configuration to use. Defaults to the default cluster configuration.",
+			),
+		},
+	}
+}
+
+// OptionalRegionAndProfileFlags provides these flags:
+// - OptRegionFlag inline overrides region
+// - OptClusterConfigFlag specifies the cluster profile to read from config
+// - OptECSProfileEnvVar specifies the credentials profile to read from the config
+// - OptAWSProfileFlag specifies the AWS Profile to use for credential information
+func OptionalRegionAndProfileFlags() []cli.Flag {
```

Comment:
This is a bit odd because it provides 4 optional flags while the name only suggests two. Is it possible to rename the function?

Reply:
I believe this name is my fault... I think the original idea was that cluster config, ecs profile, and aws profile are all "profiles".

Reply:
Yes, I didn't want to touch that method for the time being in case it affects the behavior of commands not related to local subcommands, but i created an issue here: https://github.com/aws/amazon-ecs-cli/issues/877